805-airgap.md

```diff
@@ -52,11 +52,13 @@ Archiving a CNAB and its images at a point of time as a CNAB Thick Bundle
 provides protection against modification or deletion of images and also provides a central location for code
 auditing and digital forensics of all code and references used in the CNAB.
 
-## Relocation
+## Image Relocation
 
-When the images of a CNAB are placed in a private registry, the bundle needs to be modified so that invocation
-images are downloaded from the private registry before execution and so that the invocation
-images are aware of the original and new values of image references.
+When the images of a CNAB are _relocated_ to (that is, stored in), a private registry, the images should be loaded from the private registry when they are run.
+This ensures that the repositories hosting the original image references need not be available.
```

Comment:
Tiniest of nits: "ensures that... need not be available" feels awkward. Maybe "ensures that CNAB operations can function properly even if the original image repository is unavailable" or something like that.

Reply:
It was awkward. Fixed. Thanks.